requirements.txt

```diff
@@ -6,3 +6,5 @@ pandas
 scikit-optimize
 ipykernel
 matplotlib
+xarray
+dask
```

Comment:
To use [xarray.open_zarr](https://docs.xarray.dev/en/stable/generated/xarray.open_zarr.html), [zarr](https://zarr.readthedocs.io/en/stable/installation.html#installation) should be installed too, right? :thinking: